src/elements/file-selector/file-selector/readme.md

```diff
@@ -47,22 +49,12 @@ in the next example, only images are allowed.
 <sbb-file-selector accept=".png,.jpg,.jpeg"></sbb-file-selector>
 ```
 
-### Disabled
```

Comment:
Was it on purpose to remove the disabled documentation?

Reply:
I moved it under ## States

Reply:
haha I did look for it, but apparently not good enough :D